### PR TITLE
dashboard: Update to TiDB Dashboard v2021.05.12.1

### PR DESCRIPTION
**Pull request by shhdgit**
Signed-off-by: Suhaha jklopsdfw@gmail.com

### What problem does this PR solve?

Update to TiDB Dashboard v2021.05.12.1

### Check List

Tests

- Manual test (add detailed scripts or steps below)

Related changes

- Need to cherry-pick to the release branch

### CODE REVIEW

**Review by breeswish — APPROVED** [review]

**Review by nolouch — APPROVED** [review]
lgtm

File: go.mod [hunk 1]
```diff
@@ -12,12 +12,15 @@ require (
 	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
 	github.com/docker/go-units v0.4.0
 	github.com/go-echarts/go-echarts v1.0.0
+	github.com/go-openapi/spec v0.20.3 // indirect
```
Comment (HunDunDM) [inline go.mod:15]: Can you try `go get -d github.com/pingcap/tidb-dashboard@v2021.05.12.1 && make pd-server && go mod tidy` ? I see some `indirect`.

Comment (shhdgit) [reply go.mod]: `go get -d` was awesome!

**Review by HunDunDM — APPROVED** [review]

**Pull request merged**